tool/thingtalk-dataset.js

```diff
@@ -0,0 +1,312 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// A tool to manipulate ThingTalk datasets
+//
+// Copyright 2019 National Taiwan University
+//
+// Author: Elvis Yu-Jing Lin <r06922068@ntu.edu.tw> <elvisyjlin@gmail.com>
+//
+// See COPYING for details
+"use strict";
+
+const stream = require('stream');
+const ProgressBar = require('progress');
```

Comment:
We already have a ProgressBar implementation in the library, do you need a new one?

Reply:
Where is our implementation?

Reply:
Sorry for my stupid question, I found it in `lib/progress_bar`. Thanks!